test/pull-request-hunks.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {expect} from 'chai';
+import {describe, it, before, afterEach} from 'mocha';
+import {setup} from './util';
+import * as sinon from 'sinon';
+import {Octokit} from '@octokit/rest';
+import {getPullRequestHunks} from '../src/github-handler/comment-handler/get-hunk-scope-handler/remote-patch-ranges-handler';
+
+before(() => {
+  setup();
+});
+
+describe('getPullRequestHunks', () => {
+  const upstream = {owner: 'upstream-owner', repo: 'upstream-repo'};
+  const pullNumber = 10;
+  const pageSize = 80;
+  const octokit = new Octokit({});
+  const sandbox = sinon.createSandbox();
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it('Returns the correct values when octokit and patch text parsing function execute properly', async () => {
+    // setup
+    const patch = `@@ -1,2 +1,5 @@
+ Hello world
+-!
++Goodbye World
++gOodBYE world
++
++Goodbye World`;
+    const listFilesOfPRResult = {
+      headers: {},
+      status: 200,
+      url: 'http://fake-url.com',
+      data: [
+        {
+          sha: 'a1d470fa4d7b04450715e3e02d240a34517cd988',
+          filename: 'Readme.md',
+          status: 'modified',
+          additions: 4,
```

Comment:
is there any value in pulling this into a fixture? maybe not worth it if we're only going to have one smoke test, but if we're going to add a few payloads like this over time, perhaps we could collect them in a folder.